### PR TITLE
Update git.md

### PR DESCRIPTION
**Pull request by herronk2**

### CODE REVIEW

File: docs/services/git.md [hunk 1]
```diff
@@ -8,15 +8,8 @@ Redbrick uses [Gitea](https://gitea.io/en-US/) a community driven fork of
...
-We run Gitea in Docker on zeus. We run a postgres database in a second container
-for storage of users and repo information. Gitea stores its main conf in
-`/var/git/gitea/app.ini` in a volume.
-
-See
-[docker-sevices repo](https://github.com/redbrickCmt/docker-compose-services)
-for configs. The Main points of this conf the SSH ports and login. SSH is set to
-port `10022` due to ssh on zeus using port `22`. We disable openid login and
-user registration to stop none redbrick users joining.
+We run Gitea on daedalus which runs NixOS. The config can be found [here](https://git.redbrick.dcu.ie/m1cr0man/nix-configs-rb/src/branch/master/services/gitea.nix).
```
Comment (m1cr0man) [inline docs/services/git.md:11]: Nope, gitea runs on hardcase.

**Review by m1cr0man — CHANGES_REQUESTED** [review]
Wrong host

**Comment by theycallmemac** [comment]
@herronk2 if you are at hackerclub this wednesday we can update your PR - otherwise I'll close this and get someone else to make the change!

**Comment by m1cr0man** [comment]
Hey I have totally overhauled the git docs now and they reflect the current state. This PR is superseded now.

**Pull request closed without merging**